pkgs/development/libraries/sqlite/default.nix

```diff
@@ -41,6 +41,9 @@ stdenv.mkDerivation rec {
     "-DSQLITE_SECURE_DELETE"
     "-DSQLITE_MAX_VARIABLE_NUMBER=250000"
     "-DSQLITE_MAX_EXPR_DEPTH=10000"
+    # Release notes for 3.29.0 say this is recommended
+    # https://www.sqlite.org/compile.html#rcmd
+    "-DSQLITE_DQS=0"
```

Comment:
I wonder about the comment here (should go into the commit message rather) since our settings don't follow all the recommendations, for instance they recommnad DSQLITE_MAX_EXPR_DEPTH=0 Shouldn't we update others as well  ?

Reply:
Good question, the reasoning for these particular settings (where deviating from upstream) is unknown :/.  Perhaps answers could be found with some git history spelunking but even better would be if we documented the purpose/scope/validation-still-needed (perhaps these parameters help one guy's use case but are terrible for other folks'? who can say....).

In short I suppose we should push the technical debt further down the road and leave the options as-is? :(

Reply:
If we change it then better early than right before the release.
SQLITE_MAX_VARIABLE_NUMBER and SQLITE_MAX_EXPR_DEPTH can be explained via https://github.com/NixOS/nixpkgs/pull/36293 .
As for other recommended defaults. Maybe we could just add them one by one across updates in order not to break too many things ?